packages/legend-application-studio/src/components/editor/side-bar/testable/GlobalTestRunner.tsx

```diff
@@ -429,14 +429,14 @@ export const GlobalTestRunner = observer(
     const editorStore = useEditorStore();
     const globalTestRunnerState = props.globalTestRunnerState;
     const isDispatchingAction = globalTestRunnerState.isDispatchingAction;
+
+    const extraTestRunnerTab = editorStore.pluginManager
```

Comment:
probably rewrite this to the more idiomatic form like what we do in `DataSpaceExecutableTDSResultView`